src/test/java/org/sonatype/cs/metrics/model/DbRowTest.java

```diff
@@ -0,0 +1,55 @@
+package org.sonatype.cs.metrics.model;
```

Comment:
I think for this file, we need only to test the `toString()` override.

Reply:
I think we can cover this under the same changes for DbRowStrTest.java